modules/ROOT/pages/mule-4-api-javadoc.adoc

```diff
@@ -4,6 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: java, classes, javadoc, reference, objects, methods
 
+This is the Mule API documentation generated by Javadoc. It contains definitions for classes, fields and their data types, and also describes methods including their usage, parameters and return values.
```

Comment:
Good start. Thanks. I am wondering a few things. What is the API for? Who uses it? When would they use it? Can they use something else for some of this? Do you think we can get some answers to these questions?

Reply:
Sure, I'm working on this.